test/test_interpolation_data.py

```diff
@@ -346,14 +347,16 @@ def test_calculate_fine_phonons_no_asr(self):
                             self.expctd_freqs_no_asr.to('hartree').magnitude,
                             atol=1e-10)
 
-    def test_calculate_fine_phonons_no_asr_c(self):
-        self.data.calculate_fine_phonons(self.qpts, use_c=True)
+    @parameterized.expand([(True, ), (False, )])
+    def test_calculate_fine_phonons_no_asr_c(self, test_c: bool):
+        self.data.calculate_fine_phonons(self.qpts, use_c=test_c, fall_back_on_python=not test_c)

```

Comment:
Does `@parameterized.expand([(True, ), (False, )])` in this case mean it will test with both `use_c=True` and `False`? A few lines above we have `test_calculate_fine_phonons_no_asr` which tests with `use_c=True` so we're duplicating tests.

Maybe just use `@parameterized.expand([(True, )])` for now and we can combine the `<test>`, `<test>_c` and `<test>_c_2threads` into one test in another issue?

The same goes for all the other `@parameterized`

Reply:
My mistake, I misunderstood the purpose of the other tests! But yes it does mean what you have said. I think for now I will remove parameterized completely, because pytest has another method for parameterisation so, for now, it makes sense not to use it when needed.